acs-engine.go

```diff
@@ -122,6 +163,8 @@ var noPrettyPrint = flag.Bool("noPrettyPrint", false, "do not pretty print outpu
 var artifactsDir = flag.String("artifacts", "", "directory where artifacts will be written")
 var classicMode = flag.Bool("classicMode", false, "enable classic parameters and outputs")
 var parametersOnly = flag.Bool("parametersOnly", false, "only output the parameters")
+var caCertificatePath = flag.String("caCertificatePath", "", "the path to the CA Certificate file")
```

Comment:
So what are we doing instead? Always treating and storing apimodel + caCertificate separately?